### ISSUE TITLE
Forbidden characters can be entered in the photo inventory autosave path

### ISSUE THREAD

**Issue by Wanduoying** [Yellow-Dog-Man/Resonite-Issues#2312]
### Describe the bug?

Some characters cannot be used in inventory folder names. (for example, "double quotation marks"). However, you can enter those characters in the photo inventory autosave path.

### To Reproduce

No response

### Expected behavior

Refuse to accept forbidden characters. Or give a visual warning.

### Screenshots

![Resonite01](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/35964779/05df10d9-47d7-4b33-a05f-0e00f45e983b)
![Resonite02](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/35964779/708445bc-e855-4eec-b099-53a888051225)

### Resonite Version Number

2024.6.11.74

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Meta Quest 3, Desktop

### Log Files

[DESKTOP-N94K8PD - 2024.6.11.74 - 2024-06-13 09_43_35.log](https://github.com/user-attachments/files/15812324/DESKTOP-N94K8PD.-.2024.6.11.74.-.2024-06-13.09_43_35.log)

### Additional Context

An inaccessible directory was created and screenshots stored in that folder were taking up space.

### Reporters

Wanduoying / @Wanduoying

**Comment by shiftyscales** [comment]
Thanks for the report, @Wanduoying. This issue has some overlap with #1799 in part- it shows that certain characters are already stripped e.g. frontslash `/`, but it seems others are not.

Seeking input from @ProbablePrime.

If you need to remove records from the inaccessible directory in the interim, you can send the `/requestRecordUsageReport` command to the Resonite account on your contacts list to get a list of records and use the `/deleteRecord <record ID>` command to attempt to delete any of those records.

Additional detail on the usage of these commands can be found on the wiki: https://wiki.resonite.com/Resonite_Bot

Please let me know if that doesn't work.

**Comment by ProbablePrime** [comment]
Yeah ill grab this when i can

**Comment by ProbablePrime** [comment]
@DanteTucker With your recent change to the exporter, it feels like you might be able to handle this one as well. Would you like to take it?

**Comment by DanteTucker** [comment]
Sure. I had a preliminary look at this and I can't find a good way to provide errors/feedback to the user however without big changes to to the settings system, it just doesn't support that.

I can at most just filter out invalid path characters with no explanation. Having inventories fill up with ghost items is bad enough to just start with that minimum sanity check I suppose.

**Comment by Frooxius** [comment]
If it would help, I could add a mechanism for the feedback or provide you with guidance on how to do that.

**Comment by DanteTucker** [comment]
> If it would help, I could add a mechanism for the feedback or provide you with guidance on how to do that.
> 
> I'd avoid doing "silent" filtering, because that can itself make some issues fester.

Yeah I'm just not sure where this would be. My solution for the exporter dialog (shoving it in the submit button) felt jank enough but the settings has even less options for displaying this info to users.
Ideally any given field would have some method of notifying the user their input is invalid.
